lib/config.js

```diff
@@ -7,7 +7,12 @@ var autoconfig = function(overrides) {
     IP: process.env.OPENSHIFT_NODEJS_IP || '0.0.0.0',
     PORT: process.env.FH_PORT || process.env.OPENSHIFT_NODEJS_PORT || 8001,
     dataTopicPrefix: ':cloud:data',
-    persistentStore: process.env.WFM_USE_MEMORY_STORE === "true"
+    persistentStore: process.env.WFM_USE_MEMORY_STORE === "true",
+    syncOptions: {
+      "sync_frequency" : 5,
```

Comment:
[Suggestion] Can we change this to 10 or 20 seconds to make development logs less verbose?  Seems like overkill for sample app.